core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -284,6 +284,16 @@ package object config {
       .intConf
       .createWithDefault(64)
 
+  private[spark] val DISK_STORE_BLACKLIST_TIMEOUT =
```

Comment:
these need to be documented unless we have a reason to not?

Reply:
Yes, I think it's nice to add description for this ConfigEntry, I will update.

Reply:
the formal documentation goes in the file docs/configuration.md.  Those are the ones that show up  here http://spark.apache.org/docs/latest/configuration.html.
We do leave some configs out of there if we think they are really advanced or users shouldn't be changing them but these seems like ones users may want to set

Reply:
done